Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -1720,6 +1722,40 @@ namespace xr
 
                                     inputSource.HandJoints[i].PoseRadius = joint.radius;
                                     inputSource.HandJoints[i].PoseTracked = (joint.locationFlags & RequiredFlags) == RequiredFlags;
+
+                                    // Rotate the orientation 90 degrees clockwise about the Y axis to convert from OpenXR to WebXR orientation
+                                    {
+                                        XrQuaternionf newRotation{};
+                                        XrQuaternionf rotationDelta{};
+
+                                        rotationDelta.w = -float(sin(M_PI/4));
+                                        rotationDelta.x = 0;
+                                        rotationDelta.y = -rotationDelta.w;
+                                        rotationDelta.z = 0;
+
+                                        newRotation.w = joint.pose.orientation.w * rotationDelta.w -
+                                                        joint.pose.orientation.x * rotationDelta.x -
+                                                        joint.pose.orientation.y * rotationDelta.y -
+                                                        joint.pose.orientation.z * rotationDelta.z;
+
+                                        newRotation.x = joint.pose.orientation.w * rotationDelta.x +
+                                                        joint.pose.orientation.x * rotationDelta.w +
+                                                        joint.pose.orientation.y * rotationDelta.z -
+                                                        joint.pose.orientation.z * rotationDelta.y;
+
+                                        newRotation.y = joint.pose.orientation.w * rotationDelta.y -
+                                                        joint.pose.orientation.x * rotationDelta.z +
+                                                        joint.pose.orientation.y * rotationDelta.w +
+                                                        joint.pose.orientation.z * rotationDelta.x;
+
+                                        newRotation.z = joint.pose.orientation.w * rotationDelta.z +
+                                                        joint.pose.orientation.x * rotationDelta.y -
+                                                        joint.pose.orientation.y * rotationDelta.x +
+                                                        joint.pose.orientation.z * rotationDelta.w;
```

Comment:
This feels like the kind of thing that we should have a library for already...

If not, consider pulling this out into a helper header